controller/space.go

```diff
@@ -65,14 +67,32 @@ func (c *SpaceController) Create(ctx *app.CreateSpaceContext) error {
 	}
 
 	err = transaction.Transactional(c.app, func(tr transaction.TransactionalResources) error {
-		// Create space resource which will represent the keyclok resource associated with this space
+		// Create space resource which will represent the keycloak resource associated with this space
```

Comment:
I think this should probably be moved into a standalone service:

- There is a significant amount of business logic here now (coordination of multiple service calls)
- We should be wrapping this operation in a single transaction, this won't work using transaction.Transactional but will work when using the service framework's internal transaction management.

Reply:
We will re-write this controller anyway when switching from Keycloak to AuthZ completely but I think it's a good idea create a space service right now and move all new business logic to that space. But keeping deprecated business logic and the new one in separate controllers/services/transactions is OK IMO...
But anyway let me see what we can do now.

Reply:
I've created a new service.

Reply:
I'm still a little concerned that we have two separate transactions in a single operation.  What happens if the second transaction fails?  We don't seem to have any facility for rolling back the first transaction, and if the `space_resources` record is successfully created the first time yet the second transaction fails, subsequent attempts will also fail because of the unique `space_resources_space_id_idx` constraint.  If the continued usage of `space_resources` is only temporary, then this might be acceptable in the meantime - what are the long term plans for this table?

Reply:
The only information stored in space_resources table is IDs of the remote Keycloak resources by space IDs. Why would we need to keep them when we get rid of KC?

Reply:
Sorry, my bad! I was thinking about the 'space' table in wit. :)